bundles/aem-modernize-tools/src/main/java/com/adobe/aem/modernize/structure/impl/StructureRewriteRuleServiceImpl.java

```diff
@@ -101,6 +96,14 @@ public void unbindPageRule(PageStructureRewriteRule rule) {
         return templates;
     }
 
+    @Override
+    public Set<String> getEditableTemplates(String staticTemplate) {
+        return pageRules.stream()
+                .filter(rule -> rule.getStaticTemplate().equals(staticTemplate))
+                .map(PageStructureRewriteRule::getEditableTemplate)
+                .collect(Collectors.toSet());
```

Comment:
did you consider orderable set to present results always in the same order?

Reply:
I don't think the order is particularly sensitive in this context. Users can always change the ordering of rows using the table controls in the UI.